execution-environment.yml

```diff
@@ -28,17 +33,15 @@ dependencies:
       - name: kubevirt.core
   system: |
     git-core [platform:rpm]
-    python3.9-devel [platform:rpm compile]
+    python3.11-devel [platform:rpm compile]
+    python3.11-rpm [platform:rpm epel]
     libcurl-devel [platform:rpm compile]
     krb5-devel [platform:rpm compile]
     krb5-workstation [platform:rpm]
-    subversion [platform:rpm]
-    subversion [platform:dpkg]
     git-lfs [platform:rpm]
     sshpass [platform:rpm]
     rsync [platform:rpm]
     epel-release [platform:rpm]
-    python-unversioned-command [platform:rpm]
```

Comment:
This package always links to `python3.9` on el9. Replaced with `alternatives` commands below.